Frameworks/Foundation/NSCFArray.mm

```diff
@@ -84,7 +84,7 @@ - (NSUInteger)count {
 - (id)objectAtIndex:(NSUInteger)index {
     if (index >= CFArrayGetCount((CFArrayRef)self)) {
         [NSException raise:@"Array out of bounds"
-                    format:@"objectAtIndex: index > count (%d > %d), throwing exception\n", index, CFArrayGetCount((CFArrayRef)self)];
+                    format:@"objectAtIndex: index > count (%lu > %lu), throwing exception\n", (unsigned long)index, (unsigned long)CFArrayGetCount((CFArrayRef)self)];
```

Comment:
We can say with confidence that if the rest of the code is not 64 bit compliant, the formatting statements now sure are.